engines/kyra/gui/saveload_eob.cpp

```diff
@@ -85,6 +85,8 @@ Common::Error EoBCoreEngine::loadGameState(int slot) {
 		in.read(c->level, 3);
 		for (int ii = 0; ii < 3; ii++)
 			c->experience[ii] = in.readUint32BE();
+		if (header.version >= 22)
+			c->hitPointsDividend = in.readSint16BE();
```

Comment:
If you write it after hitPointsMax, you also have to read it at that position?

Reply:
Good catch, corrected.